bigbluebutton-html5/imports/ui/components/actions-bar/create-breakout-room/component.jsx

```diff
@@ -210,16 +219,24 @@ class BreakoutRoom extends PureComponent {
   }
 
   componentDidMount() {
-    const { isInvitation, breakoutJoinedUsers } = this.props;
+    const {
+      isInvitation, breakoutJoinedUsers, getLastBreakouts, groups,
+    } = this.props;
     this.setRoomUsers();
     if (isInvitation) {
       this.setInvitationConfig();
-    }
-    if (isInvitation) {
```

Comment:
For clarity sake - was this check out of place? I don't spot new logic replacing it

Reply:
There were two identical conditions... so I merged them to improve readability.